src/interfaces.ts

```diff
@@ -8,10 +8,12 @@ export class ControllerAccount {
   authority!: PublicKey;
   redeemableMint!: PublicKey;
   redeemableMintDecimals!: number; // u16
+  isFrozen!: boolean;
   redeemableGlobalSupplyCap!: BN; // u128
   redeemableCirculatingSupply!: BN; // u128
+  registeredMercurialVaultDepositoriesCount!: number; // u8
+  registeredCredixLpDepositoriesCount!: number; // u8
```

Comment:
same order as controller.rs

Reply:
Is it this controller version? (the one in the main branch)

https://github.com/UXDProtocol/uxd-program/blob/a78cff4426625627e10a21ca9d7887c3cbcd6131/target/idl/uxd.json#L2016

Reply:
yeah this is the controller account's fields in order they are declared in the IDL

Reply:
Ok 👌  adding the registered depositories fields

Reply:
Done!